cvmfs/server_tool.cc

```diff
@@ -42,7 +42,7 @@ bool ServerTool::InitDownloadManager(const bool follow_redirects,
 }
 
 bool ServerTool::InitVerifyingSignatureManager(
-    const std::string &pubkey_path, const std::string &trusted_certs) {
+    const std::string &pubkey_path, const std::string &/*trusted_certs*/) {
```

Comment:
Why not remove the second argument alltogether? There's no interface to fullfill here, right?

Reply:
That's right, we'll actually do it with #3538 but we can do it here, too.